Tests/SourceKittenFrameworkTests/ClangTranslationUnitTests.swift

```diff
@@ -30,8 +30,14 @@ class ClangTranslationUnitTests: XCTestCase {
     }
 
     private func compare(clangFixture fixture: String) {
-        let unit = ClangTranslationUnit(headerFiles: [fixturesDirectory + fixture + ".h"],
-                                        compilerArguments: ["-x", "objective-c", "-isysroot", sdkPath(), "-I", fixturesDirectory])
+        let path = URL(fileURLWithPath: fixturesDirectory + fixture + ".h")
```

Comment:
This pattern is used a lot now: ` URL(fileURLWithPath: ...).standardizedFileURL.withUnsafeFileSystemRepresentation { String(cString: $0!) }`

Can you add some sort of shared helper for this? Open to API design ideas, but as a strawman: `FileUtils.unsafeSystemPath(for path: String) -> String`

Reply:
I don't think that the `unsafeSystemPath` makes sense, perhaps `FileUtils.nativePathRepresentation`?  The `withUnsafeFileSystemRepresentation` is because the closure receives a `UnsafePointer<CChar>`, but we are transacting in `String` at this layer.